x-pack/legacy/plugins/index_management/public/services/api.ts

```diff
@@ -46,6 +49,16 @@ export const getHttpClient = () => {
   return httpClient;
 };
 
+let savedObjectsClient: any;
```

Comment:
I would prefer the savedObjectClient to be in a separate file as it does not interact with the API layer. Also, see my comments about Kibana saved object "index patterns" to see if it is actually needed.

Reply:
this has been removed